src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/LicenceSummary.kt

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDateTime
+
+@Schema(description = "Describes a licence within this service")
+data class LicenceSummary(
+
+  @Schema(description = "Unique identifier for this licence within the service", example = "99999")
+  val id: Long = -1,
+
+  @Schema(description = "The licence type", example = "AP")
+  val licenceType: LicenceType,
+
+  @Schema(description = "The policy version number the licence is currently on", example = "2.1")
+  val policyVersion: String,
+
+  @Schema(
+    description = "The version of this specific licence, this is unique within the context of a booking",
+    example = "1.4",
+  )
+  val version: String,
+
+  @Schema(description = "The current status code for the licence", example = "IN_PROGRESS")
+  val statusCode: LicenceStatus,
+
+  @Schema(description = "The prison identifier for the person on the licence", example = "A1234AA")
+  val prisonNumber: String,
+
+  @Schema(description = "The prison internal booking ID for the person on the licence", example = "989898")
+  val bookingId: Long,
+
+  @Schema(description = "The case reference number (CRN) for the person on the licence", example = "X12444")
+  val crn: String,
+
+  @Schema(
+    description = "The username of the person who approved the licence on behalf of the prison governor",
+    example = "X33221",
+  )
+  val approvedByUsername: String,
```

Comment:
can't remember if we were returning only active licences or not. If not then approvedBy fields could be null